crypto/blake2/build.info

```diff
@@ -0,0 +1,3 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=\
+        blake2b.c blake2s.c {- $target{blake2b_obj} -}
```

Comment:
I changed $target{blake2b_obj} to $target{blake2_obj}.  Is there another filename that is wrong?  The name blake2_obj looks correct to me for projects that do not yet include assembly code files, which will change as soon as I am allowed to check in the SSE optimized versions down the road.  Maybe I don't understand the _asm_ variables.